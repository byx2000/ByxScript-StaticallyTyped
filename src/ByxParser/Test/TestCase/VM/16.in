int g1;
int g2;
int g3;
int g4;

function void main()
{
	int i = 100;
	int s = 0;
	while (i)
	{
		s = s + i;
		i = i - 1;
	}
	g1 = s;
	
	
	i = 2378;
	s = 0;
	while (i)
	{
		s = s + i;
		i = i - 1;
	}
	g2 = s;
	
	i = 100;
	s = 0;
	while (i)
	{
		if (i - 78)
		{
			s = s + i;
		}
		i = i - 1;
	}
	g3 = s;
	
	i = 101;
	s = 0;
	while (i)
	{
		i = i - 1;
		if (i - 78)
		{
			s = s + i;
		}
	}
	g4 = s;
}