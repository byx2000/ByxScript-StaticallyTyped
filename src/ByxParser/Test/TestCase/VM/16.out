i 5050
i 2828631
i 4972
i 4972